docs/components/ToastStackView.jsx

```diff
@@ -0,0 +1,267 @@
+import * as React from "react";
+
+import {Button, FlexBox, ItemAlign, SegmentedControl, ToastStack, ToastType} from "src";
+import Example, {CodeSample, ExampleCode} from "./Example";
+import PropDocumentation from "./PropDocumentation";
+import View from "./View";
+
+import "./ToastStackView.less";
+
+const cssClass = {
+  CONFIG_CONTAINER: "ToastStackView--configContainer",
+  CONFIG_OPTIONS: "ToastStackView--configOptions",
+  CONFIG: "ToastStackView--config",
+  CONFIG_TOGGLE: "ToastStackView--configToggle",
+  CONTAINER: "ToastStackView",
+  INTRO: "ToastStackView--intro",
+  PROPS: "ToastStackView--props",
+  TOAST_STACK: "ToastStackView--toastStack",
+};
+
+const messageMap = {
+  [ToastType.ERROR]: "burnt",
+  [ToastType.INFO]: "high in fiber",
+  [ToastType.PROCESSING]: "toasting...",
+  [ToastType.SUCCESS]: "ready!",
+  [ToastType.WARNING]: "slightly burnt",
+};
+
+const actionType = `{
+  text: string,
+  href?: string,
+  onClick?: Function,
+}`;
+
+export default class ToastStackView extends React.PureComponent {
+  static cssClass = cssClass;
+
+  state = {
+    durationOverride: "No override",
+    id: 0,
+    includeAction: false,
+    notifications: [],
+    showCloseButton: true,
+    toastType: ToastType.SUCCESS,
+  };
+
+  render() {
+    const {
+      durationOverride,
+      id,
+      includeAction,
+      notifications,
+      showCloseButton,
+      toastType,
+    } = this.state;
+
+    return (
+      <View
+        className={cssClass.CONTAINER}
+        title="ToastStack"
+        sourcePath="src/ToastStack/ToastStack.tsx"
+      >
+        <header className={cssClass.INTRO}>
+          <p>
+            Toast notifications can be used to provide feedback in response to user interactions.
+            The <code>ToastStack</code> renders toast notifications and handles their enter/leave
+            animations.
+          </p>
+          <CodeSample>{`
+            import {ToastStack, ToastType} from "clever-components";
+          `}</CodeSample>
+        </header>
+
+        <Example title="Basic Usage:">
+          <ExampleCode>
+            <ToastStack
+              className={cssClass.TOAST_STACK}
+              notifications={notifications}
+              setNotifications={ns => this.setState({notifications: ns})}
+              defaultNotificationDurationMS={5000}
+            />
+            <Button
+              value="Create notification"
+              onClick={() => this.setState({
+                id: id + 1,
```

Comment:
id is a little confusing here. Maybe call it something like notificationsCount or nextNotificationID?

Reply:
👍
I'll go with `nextNotificationID`